CHANGELOG.rst

```diff
@@ -18,6 +18,63 @@ Known Issues
 
 See the file named ``ISSUES`` in the root of the repository.
 
+v0.7.5
+------
+
+For details, then see the commit-messages, for highlights see below.
+
+* API Extensions
+
+  - Add helper `xnvme_nvm_compare()`
+
+    - Plus emulation via ramdisk
+
+* API Refactoring
+
+  These are the changes you need to account for if you are using the following
+  functions defined in the public xNVMe API:
+
+  - `xnvme_nvm_sanitize()`: arguments 'oipbp' and 'nodas' are now `bool`
+  - `xnvme_znd_mgmt_recv()`: argument renamed from 'action' to `zra`
+  - `xnvme_znd_mgmt_send()`: argument renamed from 'action_so' to `zsaso`
+  - `xnvme_cli_args`
+  
+    - `{clear,direct,all,save,verbose,ad,idw,idr}*`: changed to `bool`
+    - `action` renamed to `sanact`
+    - `zrms` renamed to `zsa`
+    
+  - `xnvme_pi_generate`
+  
+    - `dbuf` renamed to `data_buf`
+    - `mbuf` renamed to `md_buf`
+
+  - Renamed `enum xnvme_retrieve_opts` to `enum xnvme_retrieve_opts`
+
+* Library
+
+  - A version-script now controls library-symbols, effectively preventing
+    symbol leakage from linked libraries.
+
+* Infrastructure
+
+  - The ramdisk workflow is now running on Windows and macOS.
```

Comment:
The windows part is still in an open pull request

Reply:
thanks, fixed!